mrp_bom_note/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,5 @@
+* Oihane Crucelaegui <oihanecrucelaegi@avanzosc.es>
+* Pedro M. Baeza <pedro.baeza@serviciosbaeza.com>
+* Ana Juaristi <anajuaristi@avanzosc.es>
+* Bima Wijaya <bimajatiwijaya@gmail.com>
+* Agung Rachmatullah <agung.rachmatullah@ymail.com>
```

Comment:
Please add you as contributor.

Reply:
Done but the acute accent in my name seems to be problematic in the generated file index.html.